### PR TITLE
Review ch2

### PR DESCRIPTION
**Pull request by kbjarkefur**

### CODE REVIEW

**Comment by kbjarkefur** [comment]
@luizaandrade , I have marked each commit with the review item number so you can see what feedback it is that I am replying to.

The biggest edit is related to full disk encryption vs. file system encryption. I know how to use both, but got a bit unsure of details when comparing them head to head. I discussed this with a childhood friend who is a networks and systems engineer and was kind to help me talk it over. Took some time to explain our use case of people using personal devices of different platforms, and sharing data insecurely etc., but then we had a good discussion. I want to add him to the acknowledgements. I am interested to see what you think about this new wording. 

@bbdaniels , maybe you have opinions on this as well? it is the big edit towards the end f the chapter

**Comment by kbjarkefur** [comment]
Also, I have not addressed Fede's point about it is good to include version in a do-file as we do talk about that at multiple locations, and she said that she mostly skimmed the book, so she might just have missed it.

File: chapters/2-collaboration.tex [hunk 1]
```diff
@@ -679,42 +706,103 @@ \subsection{Ensuring privacy and security in research data}
...
+remains protected even if someone gain access to your computer.
+DIME uses VeraCrypt for encrypted folders,
 our protocols are available as part of the DIME Research Standards.\sidenote{
-  \url{https://github.com/worldbank/dime-standards}}
+	\url{https://github.com/worldbank/dime-standards}}
+VeraCrypt is free of charge and available for Windows, MacOS and Linux.
+While some details are different across platforms,
+the encryption will be implemented the same way for all team members.
+
+Regardless if you use full disk encryption or file system encryption,
```
Comment (bbdaniels) [inline chapters/2-collaboration.tex:800]:
```suggestion
Regardless of whether you use full disk encryption or file system encryption,
```

**Review by bbdaniels — CHANGES_REQUESTED** [review]
grammar

**Pull request merged**